.github/workflows/main/sconsOptions

```diff
@@ -61,6 +61,5 @@ if sys.platform == "win32" :
 	LOCATE_DEPENDENCY_PYTHONPATH=os.path.join(os.environ["GAFFER_BUILD_DIR"], "python")
 	GLEW_LIB_SUFFIX = "32"
 	INKSCAPE = "C:\\Program Files\\Inkscape\\bin\\inkscape.exe"
-	WARNINGS_AS_ERRORS = False
```

Comment:
Looks like there's at least one more problem to fix before we can do this :

```
D:\a\gaffer\gaffer\src\GafferImage\Merge.cpp(452) : error C2220: the following warning is treated as an error
D:\a\gaffer\gaffer\src\GafferImage\Merge.cpp(452) : warning C4723: potential divide by 0
D:\a\gaffer\gaffer\src\GafferImage\Merge.cpp(452) : warning C4723: potential divide by 0
```

Reply:
Hmmm, am I being incredibly dense or is there no division of any sort in `src/GafferImage/Merge.cpp` line 452? I don't get this warning on either of my local machines. I found a [bug confirmed](https://developercommunity.visualstudio.com/t/incorrect-divide-by-0-warning-c4723/1703482) by Microsoft, but that report makes it sound like it won't kick in until we go to the next version. But their "worked in" version numbers don't quite line up either, so go figure.

I reluctantly and disabling the warning in https://github.com/GafferHQ/gaffer/pull/5277/commits/f1b7b14d9aaf127941c85018f91b98924477ccd4 but if anyone has a lead to try I'll try it out.